life_expectancy/load_save.py

```diff
@@ -0,0 +1,52 @@
+"""Module for loading and saving life expectancy data."""
+
+from pathlib import Path
+from typing import Optional, Protocol
+import pandas as pd
+from .my_constants import BASE_DIR, Country
+
+class LoadData(Protocol):
+    """ Strategy for strategies to Eurostat data of different file types """
+    def read_data(self, path: Optional[Path] = None) -> pd.DataFrame:
+        ...
+
+class LoadJson:
+    """ Strategy to load zipped json Eurostat data """
+    def read_data(self, path: Optional[Path] = None) -> pd.DataFrame:
+        if path is None:
+            path = BASE_DIR / 'eurostat_life_expect.zip'
+        print(path)    
+        return pd.read_json(path, compression = 'zip')
+    
+class LoadTsv:
+    """ Strategy to load tsv Eurostat data"""
+    def read_data(self, path: Optional[Path] = None) -> pd.DataFrame:
+        if path is None:
+            path = BASE_DIR / 'eu_life_expectancy_raw.tsv'
```

Comment:
nice to have:
- consider passing the file names has variables that you load from a settings file. Could be you `my_constants.py` or `__init__.py` like mentioned before